### PR TITLE
Component/club card

### PR DESCRIPTION
**Pull request by admtomas**
Club card created ready for inspection.

### CODE REVIEW

File: src/components/ClubCard/ClubCard.styles.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  transition: all 0.2s ease-in-out;
+  cursor: pointer;
+  &:hover,
+  &:active {
+    transform: translate(0rem, -0.7rem);
+    box-shadow: var(--card-shadow);
+  }
+  &:focus {
+    box-shadow: ${({ color }) => `0 0 0 0.3rem var(--${color})`};
+  }
```
Comment (r1903) [inline src/components/ClubCard/ClubCard.styles.js]: Do we have to pass 'color' prop to the component?

File: src/components/ClubCard/ClubCard.styles.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+`;
+
+export const Image = styled.img`
```
Comment (r1903) [inline src/components/ClubCard/ClubCard.styles.js:28]: I think we have to add padding here as Figma has a bit of padding.

**Review by r1903 — COMMENTED** [review]
Hello Tomasz,

Club card component looks good. But please check below comments.

File: src/components/ClubCard/ClubCard.styles.js (hunk 2 above)
Comment (admtomas) [reply src/components/ClubCard/ClubCard.styles.js:28]: I added 1rem padding to the container. But i must stress out that design seen on Figmas  _Prototypes and _Molecules pages are diferent. Its a confusing.

File: src/components/ClubCard/ClubCard.styles.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+export const ContentContainer = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background: var(--white);
+  border: 0.1rem solid var(--lightblue);
+  border-radius: var(--smallest-space);
+
+  max-width: 60rem;
```
Comment (r1903) [inline src/components/ClubCard/ClubCard.styles.js:13]: max-width is 32.8rem.

**Review by r1903 — APPROVED** [review]
Everything looks good (Just have look at max-width). Well done!

**Pull request merged**